internal/bminventory/inventory.go

```diff
@@ -1323,7 +1323,7 @@ func (b *bareMetalInventory) InstallClusterInternal(ctx context.Context, params
 			}
 		}()
 
-		if err = b.generateClusterInstallConfig(asyncCtx, *cluster, clusterInfraenvs); err != nil {
+		if err = b.generateClusterInstallConfig(asyncCtx, *cluster, clusterInfraenvs, b.ServiceBaseURL); err != nil {
```

Comment:
why do you need to pass `ServiceBaseURL` if it's part of the same object (`bareMetalInventory`)